webpack/assets/javascripts/react_app/components/BreadcrumbBar/BreadcrumbBarActions.js

```diff
@@ -27,6 +28,8 @@ export const removeSearchQuery = resource => dispatch => {
   loadSwitcherResourcesByResource(resource)(dispatch);
 };
 
+export const initBreadcrumbsBar = () => ({ type: BREADCRUMB_BAR_INIT });
```

Comment:
The component name is `BreadcrumbBar`.  Prefer renaming this to `initBreadcrumbBar` to remain consistent.